src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -213,7 +217,8 @@ static void wrapped_rr_closure(grpc_exec_ctx* exec_ctx, void* arg,
       grpc_grpclb_client_stats_unref(wc_arg->client_stats);
     }
     if (GRPC_TRACER_ON(grpc_lb_glb_trace)) {
-      gpr_log(GPR_INFO, "Unreffing RR %p", (void*)wc_arg->rr_policy);
+      gpr_log(GPR_INFO, "[grpclb %p] Unreffing RR %p", wc_arg->glb_policy,
```

Comment:
I think that all of the `gpr_log()` calls inside of a `if (GRPC_TRACER_ON(grpc_lb_glb_trace))` block should use `GPR_DEBUG` instead of `GPR_INFO`.

Reply:
Do we have any guideline about this? I think those concerns are independent.

Reply:
I don't think we've historically paid enough attention to this, but in general I think trace information belongs at the debug log level.  But we certainly don't need to hold up this PR for that if you don't want to.